aws/README.md

```diff
@@ -0,0 +1,55 @@
+# Production infrastructure
+
+This directory contains the configuration files for the production infrastructure of the epid project.
+
+## One-time setup
+
+Download the aws and ecs-cli tools from Amazon.
+
+Configure the aws CLI, specify `us-east-1` as the default region and enter your _Access keys_ generated from the [web console](https://console.aws.amazon.com/iam/home?region=us-east-1#/security_credentials).
+
+```shell
+aws configure
+```
+
+Configure ecs-cli to work on our cluster:
+
+```shell
+ecs-cli configure --cluster epid-dev --default-launch-type FARGATE --region us-east-1
+```
+
+Create the cluster:
```

Comment:
Er, isn't this already done? Can we cut the stuff that's the completed one-time-setup?

Reply:
No this is weird but if you look at the help for the configure command it tells you that resources aren't created until you issue the up command

```
...
   --cluster value, -c value            Specifies the ECS cluster name to use. If the cluster does not exist, it is created when you try to add resources to it with the ecs-cli up command. [$ECS_CLUSTER]
```